SITnett/graphics/hamburger.js

```diff
@@ -0,0 +1,8 @@
+function hamburgerFunction() {
+    var x = document.getElementById("menuLinks");
```

Comment:
Vi har vel ikke snakket om JS-stil, men det kan hende vi har lyst til å bruke `let` og `const` fremfor `var`, bare for å være eksplisitte på om en variabel kommer til å endres eller ikke. Det er igrunnen ikke så farlig, og vi trenger ikke gjøre noe med det – jeg ser for meg at vi ikke kommer til å skrive så mye JS for denne siden uansett.

Reply:
Ok, lar det stå for nå dette også!